front/app/containers/SearchPage/SearchPage.tsx

```diff
@@ -502,43 +502,36 @@ class SearchPage extends React.Component<SearchPageProps, SearchPageState> {
   };
 
   componentDidMount() {
-    let searchTerm = this.props.location.search
-    searchTerm = searchTerm.slice(2)
-    searchTerm = searchTerm.replace(/\+/g, " ")
-    if (this.state.showCards) {
-      window.addEventListener('scroll', this.handleScroll);
-      if(this.props.location.search){
-        let q = {key:"AND",children:[{children: [], key: searchTerm}]} 
-        this.setState({
+    let searchTerm = this.props.location.search;
+    searchTerm = searchTerm.slice(2);
+    searchTerm = searchTerm.replace(/\+/g, ' ');
+    if (this.props.location.search) {
+      let q = { key: 'AND', children: [{ children: [], key: searchTerm }] };
+      this.setState(
+        {
           params: {
             q: q,
             aggFilters: [],
-            crowdAggFilters:[],
-            sorts:[],
+            crowdAggFilters: [],
+            sorts: [],
             page: 0,
-            pageSize:25
+            pageSize: 25,
```

Comment:
pageSize: 25 appears at least 4 other places in the code base already.  It might be worth exporting this number from shared.ts rather than adding another.